src/main/java/core/basesyntax/figures/Figures.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.figures;
+
+public abstract class Figures {
+
+    protected double area;
+    protected Color color;
+
+    public abstract void displayInfo();
+
+    public abstract void draw();
```

Comment:
these methods better to move to interface and make this class implement it (interface - behaviour, abstract class state)